cmd/internal/kube/test_resource/resources.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
```

Comment:
I haven't written any unit tests to go alongside this, but it works well for interacting with the CLI